src/common.js

```diff
@@ -0,0 +1,21 @@
+/** @param {Function} RequestApiClass - Class object of the API which is to be exposed via postMessage RPC */
+async function runKeyguard(RequestApiClass) {
+
+    // expose KeyStore to mockup overwrites
+    self.KeyStore = KeyStore;
```

Comment:
`KeyStore` is already a global variable, why do you need to assign it to the window here?